themes/ohcoe/static/js/scripts.js

```diff
@@ -151,4 +169,44 @@ $(function() {
     if (!localStorage.getItem('role') && !localStorage.getItem('speciality')) {
         $('#demographic-info').show();
     }
+
+    // Render on Cumulative Review page
+    if(document.getElementById('demographic-q-responses') &&
```

Comment:
I'm seeing some bits here that could be a little clearer/tighter. For example, take a look at the number of times `localStorage.getItem('role')` is called here.

Reply:
Got it, I've refactored this.

Reply:
Nice, looks good.